Makefile

```diff
@@ -28,8 +28,8 @@ NPROCS ?= 1
 # to half the number of CPU cores.
 GO_TEST_PARALLEL := $(shell echo $$(( $(NPROCS) / 2 )))
 
-GO_REQUIRED_VERSION ?= 1.19
-GOLANGCILINT_VERSION ?= 1.53.3
+GO_REQUIRED_VERSION ?= 1.21
```

Comment:
I don't think anything reads this variable anymore. We let go.mod handle it. https://github.com/search?q=repo%3Acrossplane%2Fbuild+GO_REQUIRED_VERSION&type=code

Reply:
Roger. I'll delete it.